src/clase/Jugador.java

```diff
@@ -0,0 +1,48 @@
+package clase;
+public class Jugador {
+    private String nombre;
+    private int edad;
+    private int partidasGanadas;
+    private int rachaActual;
+
+    // Constructor
+    public Jugador(String nombre, int edad) {
+        this.nombre = nombre;
+        this.edad = edad;
+        this.partidasGanadas = 0;
+        this.rachaActual = 0;
+    }
+
+    // Getters y Setters
```

Comment:
Ya sabemos que es cada cosa no es necesario agregar comentarios
```suggestion
    public Jugador(String nombre, int edad) {
        this.nombre = nombre;
        this.edad = edad;
        this.partidasGanadas = 0;
        this.rachaActual = 0;
    }

```